docs/tutorial.rst

```diff
@@ -742,50 +742,83 @@ Working with Metadata
 Metadata is information associated with entities that tskit doesn't use or interpret,
 but which is useful to pass on to downstream analysis such as sample ids, dates etc. See
 :ref:`sec_metadata` for a full discussion. Each table has a
-``metadata_schema`` which details the
+:class:`tskit.MetadataSchema` accessed as ``table.metadata_schema``. which details the
 contents and encoding of the metadata for each row. A metadata schema is a JSON document
 that conforms to `JSON Schema <https://json-schema.org/understanding-json-schema/>`_
 (The full schema for tskit is at :ref:`sec_metadata_schema_schema`).
-Here's a simple example schema, which we'll apply to the individuals table:
+
+++++++++++++++++++++++++++++
+Reading metadata and schemas
+++++++++++++++++++++++++++++
+
+Metadata is automatically decoded using the schema when accessed via a
+:class:`TreeSequence` or :class:`TableCollection` Python API. For example:
+
+.. code-block:: python
+
+    >>> ts.individual(0).metadata
+    {'accession': 'ERS0001', 'pcr': True}
+    >>> ts.tables.individuals[0].metadata
+    {'accession': 'ERS0001', 'pcr': True}
+
+Viewing the :class:`tskit.MetadataSchema` for a table can help with understanding
+its metadata as it can contain descriptions and constraints:
 
 .. code-block:: python
 
-    raw_schema = {
-        "codec": "json",
-        "type": "object",
-        "properties": {
-            "accession": {"type": "string"},
-            "pcr": {
-                "name": "PCR Used",
-                "description": "Was PCR used on this sample",
-                "type": "boolean",
-            },
+    >>> print(ts.table_metadata_schemas.individual)
```

Comment:
Will it actually print like this, or come out as a flat dict? If not, might be worth using pprint.pprint() so that we do get the pretty-printed version

Reply:
Fixed https://github.com/tskit-dev/tskit/pull/741/commits/ecf45ee24e9da2c248e03cf0ccde5d2dbeb90fdf